files/build_templates/sonic_debian_extension.j2

```diff
@@ -117,6 +117,9 @@ sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip2 install ipaddr
 # Install Python module for psutil
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip2 install psutil
 
+# Install Python 3 library for the Docker Engine API
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install docker==4.3.1
+
```

Comment:
I see still the docker 4.1.0 here https://github.com/Azure/sonic-buildimage/blob/10ab46f7a0f489d3d68ed37fd03bc48593f0ac2c/build_debian.sh#L417, shoun't we remove it ? or replace with this ?

Reply:
Thanks. I missed that. That's the Python 2 version. For now, we should install both Python 2 and 3 versions, because I believe we are still using the Python 2 version of Ansible. I will move my line next to this one and upgrade both to version 4.3.1